ccmlib/scylla_node.py

```diff
@@ -568,7 +568,7 @@ def _update_scylla_agent_pid(self):
 
     def wait_until_stopped(self, wait_seconds=127):
         start_time = time.time()
-        wait_time_sec = 1
+        wait_time_sec = 0.1
```

Comment:
this will not wait 0.1 in each iteration it grows exponentially - its fine, maybe the correct solution is not to have the growth so high .... mac 1 sec